src/js/builtins/BundlerPlugin.ts

```diff
@@ -320,7 +320,7 @@ export function runOnLoadPlugins(this: BundlerPlugin, internalID, path, namespac
           path,
           namespace,
           // suffix
-          // pluginData
+          pluginData,
```

Comment:
@colinhacks Is this in the right order? I notice everywhere else pluginData is passed as the last argument?

I ask because I am highly interested in this PR and not sure if this was the cause of your segfaulting.